### DISCUSSION TITLE
Date input should support JDay based date inputs

### DISCUSSION THREAD

**Discussion by QGerrit** [RocketCommunicationsInc/astro#867] [Feature Requests]
Feature request:

Before Astro supported `datetime-local` as a type of input, my team built a date input picker based on JDay for choosing the date.

![image](https://user-images.githubusercontent.com/97202638/172248471-e5b1ffad-5693-42f6-8f9e-9a7b967eff09.png)

Now that `datetime-local` is supported, we'd really like to switch to using that, but our users are accustomed to dealing with JDay values rather than month/day when copying dates from an email. Is a JDay based date input on the horizon somewhere?

**Comment by micahjones13** [comment]
I've created a ticket and we will look into adding this. I'll post here with updates when I have them. Thanks for bringing this up!

**Comment by micahjones13** [comment]
Hey @QGerrit, sorry for the delay on this. I have some clarifying questions for you: 
- Is what you're looking for an input that is essentially `datetime-local`, but uses the JDay instead of MM/DD/YYYY? Or are you looking for something like the second input in your example, one that just holds the JDay? 
- Would you expect the calendar date-picker UI to pop up for JDay selections as well?

**Comment by QGerrit** [comment]
1. Yes, that's exactly what I was hoping for, we went for multiple inputs when building one for simplicity, but an input with a placeholder of `JJJ YYYY` (`YYYY JJJ`?) would be nicer from a user experience I think.
2. Ideally, yes. It might not be used all the time, but is very convenient!

**Comment by micahjones13** [comment]
Hey @QGerrit got some more clarifying questions for you -
- What type of value would you expect when this input is submitted? The JDay, or MM/DD/YYYY value converted from the JDay? Something else?

**Comment by QGerrit** [comment]
Since that's purely display information, it seems like it would be best to preserve the same output format as the normal datetime input...  however, it doesn't actually matter much to me, since we're grabbing the value and parsing it into an integer (epoch time) to be stored in the database.
